webapp/lib/lazy_list_view_model.dart

```diff
@@ -44,7 +44,7 @@ class LazyListViewModel {
       position = 0;
     }
     _items.insert(position, item);
-    if (position < _listView.children.length) {
+    if (position < _listView.children.length - 1) {
```

Comment:
This is because the `_listView.children` also includes the `_scollPad`, so it's not the same length as `_items`. Without this change, the if clause was always positive, and so new items were always being added to the DOM rather than lazily added.

Reply:
Good catch! Thanks!